weave/weave_init.py

```diff
@@ -155,3 +160,13 @@ def init_local() -> InitializedClient:
     server.setup_tables()
     client = weave_client.WeaveClient("none", "none", server)
     return InitializedClient(client)
+
+
+def wandb_run_project_matches_weave_project(weave_project: str) -> bool:
+    wandb_uri = weave_client.safe_current_wb_run_id()
+    if not wandb_uri:
+        return True
+
+    wandb_project = wandb_uri.split("/")[1].split("/")[0]
```

Comment:
This line seems strange to me. If you've split on '/' there shouldn't be another '/' in `[1]` right?

Reply:
that is broken. 🤦🏻